### PR TITLE
move inactive SIG Cloud Provider approvers to emeritus_approvers

### PR DESCRIPTION
**Pull request by andrewsykim**
Signed-off-by: Andrew Sy Kim <kim.andrewsy@gmail.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Moving SIG Cloud Provider approvers that have been inactive for >= 1 year to emeritus_approvers.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cmd/cloud-controller-manager/OWNERS [hunk 1]
```diff
@@ -3,18 +3,17 @@
...
 - thockin
 - luxas
-- wlan0
 - andrewsykim
 - cheftako
 - stewart-yu
 - sttts
-- mcrute
+emeritus_approvers:
+- wlan0
```
Comment (andrewsykim) [inline cmd/cloud-controller-manager/OWNERS:16]: cc @wlan0 https://github.com/kubernetes/kubernetes/commits?author=wlan0

File: staging/src/k8s.io/legacy-cloud-providers/OWNERS [hunk 2]
```diff
@@ -5,12 +5,12 @@ approvers:
 - dims
 - andrewsykim
 - cheftako
-- mcrute
```
Comment (andrewsykim) [inline staging/src/k8s.io/legacy-cloud-providers/OWNERS:8]: cc @mcrute https://github.com/kubernetes/kubernetes/commits?author=mcrute

File: staging/src/k8s.io/legacy-cloud-providers/aws/OWNERS [hunk 3]
```diff
@@ -2,18 +2,18 @@
...
 - gnufied
 - jsafrane
 - justinsb
-- zmerlynn
 - chrislovecnm
 - nckturner
 - micahhausler
 - m00nf1sh
+emeritus_approvers:
+- zmerlynn
```
Comment (andrewsykim) [inline staging/src/k8s.io/legacy-cloud-providers/aws/OWNERS:18]: cc @zmerlynn https://github.com/kubernetes/kubernetes/commits?author=zmerlynn

File: staging/src/k8s.io/legacy-cloud-providers/vsphere/OWNERS [hunk 4]
```diff
@@ -4,15 +4,15 @@ approvers:
 - abrarshivani
 - baludontu
 - divyenpatel
-- imkin
```
Comment (andrewsykim) [inline staging/src/k8s.io/legacy-cloud-providers/vsphere/OWNERS:7]: cc imkin

**Review by andrewsykim — COMMENTED** [review]
/hold

File: staging/src/k8s.io/legacy-cloud-providers/vsphere/OWNERS (hunk 4 above)
Comment (andrewsykim) [reply staging/src/k8s.io/legacy-cloud-providers/vsphere/OWNERS:7]: cc @imkin https://github.com/kubernetes/kubernetes/commits?author=imkin

**Review by andrewsykim — COMMENTED** [review]
/hold cancel
/priority important-longterm

Removing the hold on this, didn't get any responses for the last week.

File: staging/src/k8s.io/legacy-cloud-providers/OWNERS (hunk 2 above)
Comment (mcrute) [reply staging/src/k8s.io/legacy-cloud-providers/OWNERS:8]: I haven't had much time to dedicate to this lately so I'm okay with being removed as an active maintainer. Any reason not to just push me into emeritus instead?

Comment (andrewsykim) [reply staging/src/k8s.io/legacy-cloud-providers/OWNERS:8]: I did move you to `emeritus_approvers`, I think the way the diff is rendered makes it difficult to see that :P

Comment (mcrute) [reply staging/src/k8s.io/legacy-cloud-providers/OWNERS:8]: Oh, read the diff out of context. I hope to have more time in the future to contribute to cloud provider again. Until then...

/lgtm

**Pull request merged**